src/main.rs

```diff
@@ -0,0 +1,65 @@
+mod config;
+mod client;
+
+use std::{env, process};
+
+fn main() {
+    let cfg = match config::read() {
+        Ok(cfg) => cfg,
+        Err(_) => {
+            println!("missing config, path: {}\n
+                     rerun `lmz configure`", config::path());
+
+            process::exit(1);
+        },
+    };
```

Comment:
Using someting like `eyre` for this case, we can simplify this match arm into one error propagation using `?` and attaching two "contexts" using the `eyre::WrapErr` trait:
```suggestion
    let cfg = config::read()
        .wrap_err_with(|| format!("missing config, path: {}", config::path())
        .wrap_err("return `lmz configure`")?;
```